canvas_modules/harness/test_resources/parameterDefs/slider_paramDef.json

```diff
@@ -0,0 +1,209 @@
+{
+  "titleDefinition": {
+    "title": "Slider with Min Max"
+  },
+  "current_parameters": {
+    "slider": {
+      "value": 50,
+      "min": 10,
+      "max": 100,
+      "minLabel": "",
+      "maxLabel": "",
+      "step": 10
+    },
+    "slider_error":{
+      "value": 50,
+      "min": 10,
+      "max": 100,
+      "minLabel": "",
+      "maxLabel": "",
+      "step": 10
+    },
+    "hide": true,
+    "slider-disable": false
+  },
+  "parameters": [
+    {
+      "id": "slider-disable",
+      "type": "boolean"
+	  },
+    {
+      "id": "slider",
+      "type": "integer"
+	  },
+    {
+      "id": "slider_error",
+      "type": "object",
+      "required": true,
+      "default": {
+        "value": 50,
+        "min": 10,
+        "max": 100,
+        "minLabel": "",
+        "maxLabel": "",
+        "step": 10
+      }
+    }
+  ],
+  "complex_types": [
+    {
+      "id": "sliders",
+      "parameters": [
+        
+      ]
+    }
+  ],
+  "uihints": {
+    "id": "slider",
+    "icon": "images/default.svg",
+    "label": {
+      "default": "Slider Values"
+    },
+    "parameter_info": [
+        {
+          "parameter_ref": "slider-disable",
+          "label": {
+            "default": "Disable 'Slider'"
+          }
+        },
+        {
+          "parameter_ref": "slider",
+          "label": {
+            "default": "Slider Testing"
+          },
+          "description": {
+            "default": "Slider Testing"
+          },
+          "control": "slider",
+          "class_name": "slider-control-class"
+        },
+        {
+          "parameter_ref": "slider_error",
+          "label": {
+            "default": "Error"
+          },
+          "control": "slider",
+          "description": {
+            "default": "Slider error with range more than Max'"
+          }
+        }
+    ],
+    "complex_type_info": [
+      {
+        "complex_type_ref": "slider_error",
+        "parameters": [
+          {
+            "parameter_ref": "slider",
+            "width": 10,
+            "label": {
+              "default": "Standard"
+            },
+            "control": "slider",
+            "class_name": "slider",
+            "slider": {
```

Comment:
This doesn't follow any of our standards.  I don't really even know how this works.